src/microbe_stage/Microbe.cs

```diff
@@ -323,6 +323,12 @@ public float Radius
         }
     }
 
+    /// <summary>
+    ///   Returns a squared value of <see cref="Radius"/>.
+    /// </summary>
+    [JsonIgnore]
+    public float RadiusSquared => Radius * Radius;
```

Comment:
Given that it will be called each frame, I wonder if storing the value would not be interesting for computation time. But I can't really affirm that float * float multiplication is *that* costly, so if you feel it is better as is I have no opposition.

Reply:
The argument here is probably that caching it will be a bit tricky, It'll need to be in Membrane.cs as that's the only thing that knows when Radius becomes dirty and is recomputed. Much simpler to just go with this code for now and come back to this if it turns out to be problematic.

Reply:
Alright, didn't think of that. The argument makes sense, I'm fine with keeping it as such.